web/checks_api.go

```diff
@@ -139,7 +182,7 @@ func ApiCheckGetSettingsByIdHandler(s services.ChecksService) gin.HandlerFunc {
 // @Router /api/checks/{id}/settings [post]
 func ApiCheckCreateSettingsByIdHandler(s services.ChecksService) gin.HandlerFunc {
 	return func(c *gin.Context) {
-		id := c.Param("id")
+		clusterId := c.Param("id")
```

Comment:
I would keep this to `id`. Actually, I renamed it before in other changes to make the check selection agnostic. We might have other pages with checks, but as all of them use the same backed, we could reuse this API call

Reply:
`resourceId` will play here too, I think. Good catch!